e2e/ethereum/ethereum.go

```diff
@@ -140,21 +138,21 @@ func CreateErc20Deposit(t *testing.T, client *ethclient.Client, opts *bind.Trans
 	}
 }
 
-func CreateErc721Deposit(t *testing.T, client *ethclient.Client, opts *bind.TransactOpts, destId msg.ChainId, recipient []byte, tokenId *big.Int, contracts *utils.DeployedContracts, rId msg.ResourceId) {
+func CreateErc721Deposit(t *testing.T, client *utils.Client, destId msg.ChainId, recipient []byte, tokenId *big.Int, contracts *utils.DeployedContracts, rId msg.ResourceId) {
 	data := utils.ConstructErc721DepositData(rId, tokenId, recipient)
 
-	bridgeInstance, err := bridge.NewBridge(contracts.BridgeAddress, client)
+	bridgeInstance, err := bridge.NewBridge(contracts.BridgeAddress, client.Client)
 	if err != nil {
 		t.Fatal(err)
 	}
 
-	err = utils.UpdateNonce(opts, client)
+	err = utils.UpdateNonce(client)
 	if err != nil {
 		t.Fatal(err)
 	}
 
 	if _, err := bridgeInstance.Deposit(
-		opts,
+		client.Opts,
```

Comment:
Only passing in client.Opts because we don't use anything else from the client object. thoughts?